patches/tModLoader/Terraria/ModLoader/ModBlockType.cs

```diff
@@ -23,7 +23,11 @@ public abstract class ModBlockType : ModTexturedType, ILocalizedModType
 	/// <para/> Defaults to 0, which is <see cref="DustID.Dirt"/>. To prevent spawning any hit dust, set this to -1 instead. </summary>
 	public int DustType { get; set; }
 
-	/// <summary> The vanilla ID of what should replace the instance when a user unloads and subsequently deletes data from your mod in their save file. Defaults to <see cref="TileID.Dirt"/> (0). </summary>
+	/// <summary>
+	/// The vanilla ID of what should replace the instance when a user unloads and subsequently deletes data from your mod in their save file.
+	/// <br/><br/> <see cref="Main.tileFrameImportant"/> tiles should try to match the the layout (FrameX and FrameY values) of the fallback tile so that the resulting tiles aren't broken.
```

Comment:
`the the`, and it's `TileFrameX/Y` respectively

I'm thinking if the default being 0 could cause issues for exactly the reason outlined above. Maybe TML should handle  fallbacks differently based on `Main.tileFrameImportant`? A tileFrameImportant tile rn would fall back to 0 which would most certainly break stuff (if the original tile's frames are out of bounds of the 0 type sheet)

Reply:
So, if a frameimportant tile falls back to a non-frameimportant tile, the resulting tiles will all be framed during the process so they'll fix themselves. Even though they had invalid frameXY values for a Dirt tile, during framing those will be fixed.